docs/book/src/developer/providers/v1alpha3-to-v1alpha4.md

```diff
@@ -94,6 +90,14 @@ For a `/config` folder reference, please use the testdata in the Kubebuilder pro
 
 Provider's `/config` folder has the same structure of  `/config` folder in CAPI controllers.
 
+**Changes in the `/config/rbac` folder:**
```

Comment:
IMO Those changes should not go under "## Required kustomize changes to have a single manager watching all namespaces and answer to webhook calls"; the instruction in this paragraph should not be modified given that some of the providers already applied those changes, and it will be difficult for them to figure out the delta.

Instead, there should be a separarated paragraph `## Required kustomize changes to remove Kubeadm-rbac-proxy`